### PR TITLE
Adjust formatting for `system caches delete` option descriptions

### PR DESCRIPTION
**Pull request by toph-allen**
## Intent

Resolves two issues with the `system caches delete` documentation:

1. Options were described below the example, making them less discoverable.
2. Bullet points describing options lacked a preceding double-newline, so, while GitHub rendered the list correctly, the docs site did not. The `mkdocs` site rendered them as a single paragraph.

## Type of Change

Documentation-only.

- [ ] Bug Fix            
- [ ] New Feature       
- [ ] Breaking Change   

## Approach

N/A

## Automated Tests

N/A

## Directions for Reviewers

I checked the fix in the rendered `mkdocs` site, but it'd be good for you to do so as well. Within the `docs` folder, run `make all`, then navigate to `site/index.html` and check the output.

## Checklist

- [ ] I have updated [CHANGELOG.md](../CHANGELOG.md) to cover notable changes.
- [ ] I have updated all related GitHub issues to reflect their current state.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1081,7 +1081,15 @@ Connect server, as well as the language (`-l, --language`), version (`-V,
 delete. Deleting a large cache might take a while. The command will wait for
 Connect to finish the task.
 
-Use the dry run option (`-d, --dry-run`) to surface any errors ahead of
+Use the following parameters specify the target cache:
+
+- `language` (required) must name `R` or `Python`. It is case-insensitive.
+- `version` (required) must be a three-part version number, e.g. `3.8.12`.
+- `image-name` (optional) defaults to `Local`, which targets caches used for
+  natively-executed content. Off-host images can be specified using either the
```
Comment (aronatkins) [inline README.md:1089]: Always using four-space indentation is a good way to make sure that the different processors render the Markdown consistently.

**Review by aronatkins — APPROVED** [review]

**Comment by toph-allen** [comment]
Not sure why CI is failing, but it seems likely to be outside of the scope of this PR.

**Comment by tdstein** [comment]
I just started seeing the same error in another pull request. There may be a temporary error with the host. Let's see if it resolves itself shortly.

File: README.md (hunk 1 above)
Comment (toph-allen) [reply README.md:1089]: In this case, it was just that the list was not in its own paragraph, i.e. not separated from the previous line by an empty line. I don't think indentation would do anything here? Anyway, it seems to be rendering correctly, so I'm going to resolve this conversation.

**Comment by toph-allen** [comment]
Looks like the other issue that was breaking this is https://github.com/rstudio/vetiver-python/issues/184 — `vetiver` did not specify a `pydantic` version in its `setup.cfg` so a major `pydantic` release was breaking `vetiver`.

**Review by tdstein — APPROVED** [review]

**Comment by toph-allen** [comment]
Fixed CI failures by adding `pydantic<2.0.0` to `vetiver-requirements.txt`. There is also a fix inbound in Vetiver; see https://github.com/rstudio/vetiver-python/issues/184.

**Pull request merged**